lib/OpenQA/Script/CloneJob.pm

```diff
@@ -49,6 +49,7 @@ sub is_global_setting {
 sub clone_job_apply_settings {
     my ($argv, $depth, $settings, $options) = @_;
 
+    $ENV{MOJO_LOG_LEVEL} = 'info' if $options->{verbose};
```

Comment:
I'm not sure the `-vv` bit is trivial to add, since it would touch both script and lib code, if we want to use it elsewhere. I'll just make it *debug* for now.